### DISCUSSION TITLE
Outdated milestones

### DISCUSSION THREAD

**Discussion by luigifab** [OpenMage/magento-lts#1833] [Questions & Answers] [answered]
Is someone can close our outdated [milestones](https://github.com/OpenMage/magento-lts/milestones)? 😸

**Comment by luigifab** [comment]
I suggest also to close all PR from https://github.com/OpenMage/magento-mirror, to close wiki, to update description and link to OpenMage/magento-lts repository, to drop? [pr/magento-1.9.3.2](https://github.com/OpenMage/magento-mirror/tree/pr/magento-1.9.3.2), and to archive the repository.

**Reply by fballiano** [reply]
> I suggest also to close all PR from https://github.com/OpenMage/magento-mirror, to close wiki, to update description and link to OpenMage/magento-lts repository, to drop? [pr/magento-1.9.3.2](https://github.com/OpenMage/magento-mirror/tree/pr/magento-1.9.3.2), and to archive the repository.

I didn't really understand this part

**Reply by Sdfendor** [reply]
Which part? As far as I understand it, "magento-mirror" is a project mirroring the original Magento CE which no longer is updated for quite some time. So the project can be archived. The 6 open PRs subsequently aren't relevant anymore so they can be closed and the wiki is no longer necessary there.

**Reply by fballiano** [reply]
I think only @Flyingmana can do that

**Reply by Flyingmana** [reply]
sorry, that needs a level above me :/ I dont have setting access on the *magento-mirror* repository

**Reply by fballiano** [reply]
somebody has a higher privilege? 😱🤯

**Reply by Flyingmana** [reply]
yeah, people tend to ignore it when Iam saying Iam not an owner/admin of the project/organisation 😅 Iam just one of many maintainers.

**Reply by addison74** [reply]
Personally I was left with the impression that @Flyingmana is the owner. I admit it's a surprise now to hear that he's just a maintainer. In this case I think we should know who this owner is. At least say Hello.

**Reply by Flyingmana** [reply]
@LeeSaferite did start the organisation, as can also be seen indirectly in https://github.com/OpenMage/magento-mirror/commits/magento-1.9?after=98da842ef7aa59b8b8fee642de8b0b6deec55c31+104&branch=magento-1.9&qualified_name=refs%2Fheads%2Fmagento-1.9

and @drobinson is the other owner level one.

**Reply by addison74** [reply]
Thank you for your clarifications. I haven't seen any posts of these people in the last two years. Are they still active?

**Reply by Flyingmana** [reply]
they are still available when absolutely needed. And as owners they carry the important part of legal responsibility for Org and Website (which I am honestly very happy about to not need to care about)

**Comment by fballiano** [comment, marked as answer]
> Is someone can close our outdated [milestones](https://github.com/OpenMage/magento-lts/milestones)? 😸

done
